samples/sensor/ccs811/README.rst

```diff
@@ -26,10 +30,12 @@ Building and Running on nrf52_pca20020
 Sample Output
 =============
 
-The sample below shows output from a Nordic Thingy:52 (nrf52_pca20020).
-Several bogus readings are received due to a soft reset; then there is a 4
-second startup where readings are not updated, then the sensor starts to
-emit measurements.
+The sample output below is from a `Nordic Thingy:52
+<https://www.nordicsemi.com/Software-and-tools/Prototyping-platforms/Nordic-Thingy-52>`_
+(nrf52_pca20020) that includes this sensor (and others).
+After a soft reset, there is a 5-second startup period
+where readings are unstable, and then we can see steady
+reported measurements of about 400 ppm eC02 and 0 ppb eTVOC.
```

Comment:
The changes in this file are in the wrong commit (they're not related to `drivers: sensor: ccs811: accommodate old firmware`)